build/check-images.js

```diff
@@ -137,16 +153,16 @@ function checkImageReferences(doc, $, options, { url, rawContent }) {
       let enUSFallback = false;
       if (
         !filePath &&
-        doc.locale !== "en-US" &&
-        !finalSrc.startsWith("/en-us/")
+        doc.locale !== DEFAULT_LOCALE &&
+        !finalSrc.startsWith(`/${DEFAULT_LOCALE.toLowerCase()}/`)
       ) {
-        const enFinalSrc = finalSrc.replace(
+        const enUSFinalSrc = finalSrc.replace(
           `/${doc.locale.toLowerCase()}/`,
-          "/en-us/"
+          `/${DEFAULT_LOCALE.toLowerCase()}/`
         );
-        if (Image.findByURL(enFinalSrc)) {
+        if (Image.findByURL(enUSFinalSrc)) {
           // Use the en-US src instead
-          finalSrc = enFinalSrc;
+          finalSrc = enUSFinalSrc;
```

Comment:
The above 6 lines of changes are just tidying up (refactoring). 
Instead of hardcoding the string `"en-us"` it now uses `DEFAULT_LOCALE`. 
And if the first variable is called `enUSFallback` the other one should be called `enUSFinalSrc` to match the prefixes.

Reply:
I'm always scared when I see how _fragile_ our locale casing is.